doc/whats_new.rst

```diff
@@ -114,6 +114,8 @@ BUG
 
     - Fix bug in :func:`mne.io.read_raw_edf` when ``preload=False`` and channels have different sampling rates by `Jaakko Leppakangas`_
 
+    - Fix :func:`mne.labels.read_labels_from_annot` to set label.values=1 rather than 0 for consistency with the label class by `Jon Houck`_
```

Comment:
use double-backticks on `label.values=1` to make  it code

The proper link is:
```
:func:`mne.read_labels_from_annot`
```
for bonus points you can also do
```
with the :class:`Label` class
```